samples/nrf9160/modem_trace_flash/Kconfig.trace_flash

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+module = MODEM_TRACE_FLASH_BACKEND
+module-str = Modem trace flash backend
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
+
+if NRF_MODEM_LIB_TRACE
+
+# Add NRF_MODEM_LIB_TRACE_BACKEND_FLASH to the trace backend choices.
+choice NRF_MODEM_LIB_TRACE_BACKEND
+
+config NRF_MODEM_LIB_TRACE_BACKEND_FLASH
+	bool "Stream modem traces to flash"
+	help
+	  Configure nrf_modem_lib modem traces to use the flash backend in this sample.
+
+endchoice # NRF_MODEM_LIB_TRACE_BACKEND
+
+if NRF_MODEM_LIB_TRACE_BACKEND_FLASH
+
+config PM_PARTITION_SIZE_MODEM_TRACE
+	hex "External flash space reserved for modem traces"
+	default 0x800000
```

Comment:
Could you get this from DT?

Reply:
Maybe. We have it defined in the overlay, but it's not a required property of `jedec,jesd216` (which `jedec,spi-nor` includes). So there's no guarantee it's there. The only supported board is the nrf9160DK v0.14.0+, and all newer revisions have the same 8 MB flash chip. So will leave it hard coded for now.